server/data/manageUsersApiClient.ts

```diff
@@ -9,8 +9,7 @@ export interface User {
   authSource?: string
   uuid?: string
   userId?: string
-  staffId?: number // deprecated, use userId
```

Comment:
Does this mean the wiremock stub should also be updated to remove `staffId` too? Here: https://github.com/ministryofjustice/hmpps-template-typescript/blob/92d8a3d19239a08686bb5de137bccdfe6b929c18/integration_tests/mockApis/manageUsersApi.ts#L15

Reply:
Thanks, removed from stub